cleanup/mavenSnapshotCleanupWhenRelease/MavenSnapshotCleanupWhenReleaseTest.groovy

```diff
@@ -0,0 +1,65 @@
+import static org.jfrog.artifactory.client.ArtifactoryClient.create
+
+import org.jfrog.artifactory.client.Artifactory
+
+import groovyx.net.http.HttpResponseException
+import spock.lang.Specification
+
+class MavenSnapshotCleanupWhenReleaseTest extends Specification {
+
+    def TEST_FOLDER = 'org/jfrog/test'
+    def TEST_NAME = 'test-snapshot-cleanup-when-release'
+
+    def 'Maven Snapshot Cleanup When Release test'() {
+        setup:
+        def baseurl = 'http://localhost:8088/artifactory'
+        def artifactory = create(baseurl, 'admin', 'password')
+
+        deleteNoErrorIfNotFound(artifactory, 'maven-local-lib-releases', TEST_FOLDER)
```

Comment:
@elioengcomp: I agree this is better. But the repositories where this policy is applied are defined by plugin configuration => loaded at Artifactory's startup and checked by plugin (type, handle, ...). 

I didn't found a way to inject a plugin configuration by REST API, but using [System Info](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-SystemInfo) and [Reload Plugins](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-ReloadPlugins) REST API ; perhaps `etc/plugins` could be updated dynamically (could work only for a local Artifactory).

I will have a try on this way tomorrow.